src/tlo/methods/care_of_women_during_pregnancy.py

```diff
@@ -735,6 +735,7 @@ def screening_interventions_delivered_at_every_contact(self, hsi_event):
         # Delivery of the intervention is conditioned on a random draw against a probability that the intervention
         # would be delivered (used to calibrate to SPA data- acts as proxy for clinical quality)
         if self.rng.random_sample() < params['prob_intervention_delivered_urine_ds']:
+            hsi_event.EQUIPMENT.update({'Urine dip Stick'})
```

Comment:
I dont know whats best with this equipment call considering its the same 'thing' as the consumable anyway

Reply:
Is it exact same thing as covered with consumables or an additional item used for the procedure? If the former is true, then do not add this equipment at all as it is already covered, if the latter, and it won't be used when consumable not available, the update only if it is available.

Reply:
@EvaJanouskova just following up on this since out meeting last week - i will set up an issue now and hopefully we can meet with Tim H to discuss these consumable/equipments. will leave this comment unresolved for now